src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -733,6 +739,7 @@ internal void AddResponseFileCommandsForSwitchesSinceInitialReleaseThatAreNeeded
             commandLine.AppendSwitchIfNotNull("/checksumalgorithm:", ChecksumAlgorithm);
             commandLine.AppendSwitchWithSplitting("/instrument:", Instrument, ",", ';', ',');
             commandLine.AppendSwitchIfNotNull("/sourcelink:", SourceLink);
+            commandLine.AppendSwitchIfNotNull("/langversion:", LangVersion);
```

Comment:
@jaredpar The channel passing langversion for C# actually works (`CVbcMSBuildHostObject::SetCompilerOptions` is a pass through). 
The code above does send the language version for both C# and VB, but only VB will read it at this point.